tests/tests_fabric/strategies/test_fsdp_integration.py

```diff
@@ -343,33 +344,31 @@ def test_setup_with_orig_params_and_multiple_param_groups():
         assert not isinstance(layer.weight, FlatParameter)
 
 
-@RunIf(min_cuda_gpus=2, skip_windows=True, standalone=True, dynamo=True)
+@RunIf(min_cuda_gpus=1, skip_windows=True, standalone=True, dynamo=True, min_python="3.9")
 @mock.patch.dict(os.environ, {})
-@pytest.mark.parametrize(
-    "compile_after_setup",
-    [
-        False,
```

Comment:
`compile_after_setup==False` is not a relevant parametrization because there's no interaction with fabric in that case.